src/components/BookList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookList = ({ books }) => (
+  <ul className="books">
+    {books.map((book) => (
+      <books key={book.id} title={book.title} />
```

Comment:
:red_circle: [REQUIRED]
In the this component, when mapping through the books array, use the Book component to render each book item:
```
{books.map((book) => (
  <Book key={book.id} title={book.title} author={book.author} />
))}

```